update-changelog/README.md

```diff
@@ -25,6 +25,7 @@ Next version based on the content of the top section of the changelog and the pr
 The action takes the following parameters:
 
 1. *(Required)* `changelog`: the path to the CHANGELOG.md.
+1. *(Optional)* `version`: an optional override of the version.
 1. *(Optional)* `version-suffix`: an optional suffix to append to the version.
```

Comment:
I believe this `version-suffix` is no longer needed.  One can just use the new `version` override to accomplish the same thing.